.flowconfig

```diff
@@ -9,12 +9,13 @@
 .*/staticfiles/.*
 .*/static/bundles/.*
 .*/node_modules/immer/.*.flow
+.*/node_modules/resolve/.*
```

Comment:
Apparently `resolve` intentionally includes an invali package.json (for testing purposes, I guess?) that causes problems for flow, so flowignore it.

I was surprised we don't just flowignore all of node_modules? But maybe that's normal. I haven't used flow in several years.